### DISCUSSION TITLE
Where to edit the instance name?

### DISCUSSION THREAD

**Discussion by renfie** [immich-app/immich#10129] [General]
When I receive a notification, the following link appears: http://localhost:2283...
Where can I enter the instance name so that I am also forwarded to my domain?

Sorry for the question, but I couldn't find it in the documentation or I missed it. :-)

![image](https://github.com/immich-app/immich/assets/102348785/7e1e28ee-b2c1-47dc-8248-d147dda12816)

**Comment by bo0tzz** [comment]
Admin panel > server settings > external domain

**Comment by renfie** [comment]
Oh yes. I think I was blind. Thanks for the hint. :-)
